src/actions/guides/testing/introduction.cr

```diff
@@ -0,0 +1,53 @@
+class Guides::Testing::Introduction < GuideAction
+  guide_route "/testing/introduction"
+
+  def self.title
+    "Testing Introduction"
```

Comment:
```suggestion
    "Intro to Testing"
```

To match some of the other "Intro" guides like https://luckyframework.org/guides/database/intro-to-avram-and-orms